homeassistant/components/iometer/config_flow.py

```diff
@@ -0,0 +1,94 @@
+"""Config flow for the IOmeter integration."""
+
+from typing import Any, Final
+
+from iometer import IOmeterClient, IOmeterConnectionError
+import voluptuous as vol
+
+from homeassistant.components import zeroconf
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_HOST
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA: Final = vol.Schema({vol.Required(CONF_HOST): str})
+
+
+class IOMeterConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handles the config flow for a IOmeter bridge and core."""
+
+    def __init__(self) -> None:
+        """Initialize the config flow."""
+        self.data: dict[str, Any] = {}
+
+    async def async_step_zeroconf(
+        self, discovery_info: zeroconf.ZeroconfServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle zeroconf discovery."""
+        self.data[CONF_HOST] = host = discovery_info.host
+        self._async_abort_entries_match({CONF_HOST: host})
+
+        session = async_get_clientsession(self.hass)
+        client = IOmeterClient(host=host, session=session)
+        try:
+            status = await client.get_current_status()
+        except IOmeterConnectionError:
+            return self.async_abort(reason="cannot_connect")
+
+        self.data["meter_number"] = status.meter.number
+
+        await self.async_set_unique_id(status.device.id)
+        self._abort_if_unique_id_configured()
+
+        self.context.update(
+            {
+                "title_placeholders": {
+                    "meter_number": self.data["meter_number"],
+                }
+            }
+        )
```

Comment:
```suggestion
        self.context["title_placeholders"] = {
            "meter_number": self.data["meter_number"],
        }
```

Reply:
Also, since the `flow_title` is not meant to be translated, could we set `name: f"IOmeter {self.data["meter_number"]"` instead and remove `flow_title` from strings.json?

Reply:
Yes